package.json

```diff
@@ -3,7 +3,8 @@
   "version": "1.2.0",
   "description": "Search analytics by Algolia",
   "jsdelivr": "dist/search-insights.min.js",
-  "main": "dist/search-insights.min.js",
+  "main": "dist/search-insights.cjs.min.js",
+  "browser": "dist/search-insights.min.js",
```

Comment:
I get the sense that you try avoid browser code in the node build but I think it's more important to avoid node code in the browser build, as browser build user will care more about size.

Is there a way to do this?

Reply:
`cjs` can be used both on browser and on node, but `umd` means it's definitely on browser. So, IMO, in `umd` build, we can remove node-related code. There's not much, but at least we can treeshake the part of code dealing with nodeHttp and nodeHttps.
What do you think?

Reply:
sounds good!

Reply:
https://github.com/algolia/search-insights.js/pull/140/commits/cfcbed92f329550689a552ccd47f7a614510637b

With this config, I can use `if (process.maybeNode)`. What do you think?

Reply:
https://github.com/algolia/search-insights.js/pull/140/commits/45fb4f8f69b369c6b2b075514fd1d630272a88cd

I've added `__BROWSER_BUILD__` instead of `maybeNode`.